module/chunks/chunk_assigner.go

```diff
@@ -102,7 +102,7 @@ func (p *ChunkAssigner) rngByBlockID(stateSnapshot protocol.Snapshot) (random.Ra
 		return nil, fmt.Errorf("failed to retrieve source of randomness: %w", err)
 	}
 
-	rng, err := random.NewRand(seed)
+	rng, err := random.NewChacha20PRG(seed, []byte("chunk_assign"))
```

Comment:
maybe we define "chunk_assign", "net_topology" as global constants somewhere in model files

Reply:
updated in https://github.com/onflow/flow-go/pull/1782/commits/a82b88514fab186d229fa133efa70ee46eb974f0